apps/central-scan/frontend/src/screens/settings_screen.tsx

```diff
@@ -137,8 +103,11 @@ export function SettingsScreen({
         <SignedHashValidationButton apiClient={apiClient} />
       </ButtonRow>
 
-      {isBackingUp && (
-        <Modal centerContent content={<Loading>Saving Backup</Loading>} />
+      {isSavingBackup && (
```

Comment:
https://github.com/user-attachments/assets/ba8da560-006c-4df0-8a69-c1debb175021


https://github.com/user-attachments/assets/84fd04e3-9e07-45db-94d4-bfd0e467e2aa

Reply:
Looks good! I do think the USB illustration feels a bit out of place and unnecessary, so I would just cut it

Reply:
Agree

Reply:
<img width="547" alt="Screen Shot 2024-10-14 at 3 42 59 PM" src="https://github.com/user-attachments/assets/b3399a97-14e8-4dec-86ef-57871e1aee68">
<img width="551" alt="Screen Shot 2024-10-14 at 3 43 32 PM" src="https://github.com/user-attachments/assets/d110fdad-982a-43b0-bb37-37968f7d6665">